sig-architecture/production-readiness.md

```diff
@@ -28,6 +27,19 @@ aspects of the process.
     happens if it is subsequently upgraded again?
   - Are there tests for this?
 * Scalability
+  - Does enabling/using the feautre will result in new API calls?
+    Keeping in mind the [supported limits] (e.g. 5000 nodes per cluster,
+    100 pods/s churn), what QPS is expected for each of those?
+  - Does enabling/using the feature will result in supporting new API
+    objects? How many of those will be supported (and how that translates
+    to limitations for users)?
+  - Does enabling/using the feature will result in increasing size of
+    the existing API objects? How much?
+  - Does enabling/using the feature will result in increasing latencies
```

Comment:
add "(increase time taken)" next to `latencies` to make it a bit more clear?

Reply:
@dims - not sure I understand, can you clarify?

Reply:
`will result in increasing latencies` => `will result in increasing latencies (increase time taken)` 

Just add a few more words that's all :)

Reply:
Removed the latencies - changed to "increase time take" - seems better. thanks.